src/web/features.rs

```diff
@@ -116,6 +116,24 @@ impl FeaturesPage {
     pub(crate) fn use_direct_platform_links(&self) -> bool {
         true
     }
+
+    pub(crate) fn nb_enabled_default_features(&self) -> usize {
```

Comment:
nit: what does `nb_` mean in the function name? Is there a better name?

Reply:
It means "number". Gonna find a better name.

Reply:
`count_` makes sense?

Reply:
I remove `nb_` and added `_count` at the end. But glad to see I'm not the only one with this idea. :)